packages/controller/src/resources/monitoring/tenants/grafana.ts

```diff
@@ -172,6 +169,31 @@ export function GrafanaResources(
               }
             },
             spec: {
+              initContainers: [
+                {
+                  image: "tmaier/postgresql-client",
```

Comment:
This should be moved to docker-images.json and also needs an image tag.

Reply:
Thanks. https://github.com/opstrace/opstrace/pull/278